packages/amplify-e2e-tests/src/__tests__/auth_6.test.ts

```diff
@@ -24,20 +24,41 @@ describe('zero config auth ', () => {
     deleteProjectDir(projRoot);
   });
 
-  it('...should init a javascript project and add auth with a all uptions and update front end config', async () => {
-    await initJSProjectWithProfile(projRoot, defaultsSettings);
-    await addAuthWithMaxOptions(projRoot, { useInclusiveTerminology: false });
+  it('...should init a javascript project and add auth with all options and update front end config', async () => {
+    await initJSProjectWithProfile(projRoot, defaultSettings);
+    await addAuthWithMaxOptions(projRoot, {});
     await amplifyPushAuth(projRoot);
-    let meta = getProjectMeta(projRoot);
 
-    let id = Object.keys(meta.auth)[0];
-    let authMeta = meta.auth[id];
+    const meta = getProjectMeta(projRoot);
+    const authMeta: $TSAny = Object.values(meta.auth)[1];
```

Comment:
what's the first auth resource in the list?

Reply:
the `userPoolGroups`